### PR TITLE
DPL-1116 - create rake task for printers

### PR DESCRIPTION
**Pull request by stevieing**
Closes #

#### Changes proposed in this pull request

- 

#### Instructions for Reviewers

_[All PRs] - Confirm PR template filled_  
_[Feature Branches] - Review code_  
_[Production Merges to `main`]_  
 &nbsp; &nbsp; \- _Check story numbers included_  
 &nbsp; &nbsp; \- _Check for debug code_  
 &nbsp; &nbsp; \- _Check version_

### CODE REVIEW

File: lib/tasks/create_printers.rake [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  task create: :environment do
+    [
+      { name: 'g216bc', labware_type: 'tube' },
+      { name: 'h105bc', labware_type: 'tube' },
+      { name: 'ssrtubebc-sq1', labware_type: 'tube' },
+      { name: 'aa309bc1', labware_type: 'tube' },
+      { name: 'g311bc1', labware_type: 'tube' },
+      { name: 'aa309bc3', labware_type: 'tube' }
+    ].each do |options|
+      Printer.create_with(options).find_or_create_by!(name: options[:name])
```
Comment (dasunpubudumal) [inline lib/tasks/create_printers.rake:13]: Minor thing: do we need to invoke `create_with` method here? I think `find_or_create_by` already does what `create_with` is supposed to do here.

Comment (dasunpubudumal) [inline lib/tasks/create_printers.rake:13]: Do we need to add `create_with` here? I think `find_or_create_by!(options)` would do what `create_with` is trying to do?

Comment (stevieing) [reply lib/tasks/create_printers.rake:13]: This is what we are doing elsewhere. Happy to leave it as is.

File: spec/lib/tasks/volume_tracking.rake_spec.rb [hunk 2]
```diff
@@ -22,7 +22,11 @@
       well_aliquots = create_list(:aliquot, 10, used_by: create(:pacbio_well), volume: 10, concentration: 10)
 
       # We shouldnt change the amount of aliquots
-      expect { Rake::Task['volume_tracking:clear_well_aliquot_volume'].invoke }.not_to change(Aliquot, :count)
+      expect { Rake::Task['volume_tracking:clear_well_aliquot_volume'].invoke }.not_to change(Aliquot, :count) and output(
+        <<~HEREDOC
+          -> Clearing volume of all well aliquots
+        HEREDOC
+      ).to_stdout
```
[the remark this answers is not in the document]
Comment (stevieing) [reply spec/lib/tasks/volume_tracking.rake_spec.rb:29]: This was not added by me. I just added the heredoc to capture the comment. Not planning to change it.

**Pull request merged**